Procurement/View/Converters/DivinationCardExplicitModsToFormattedRunConverter.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Windows.Documents;
+using Procurement.ViewModel;
+using System.Windows.Media;
+using System.Windows.Data;
+using System.Text.RegularExpressions;
+
+namespace Procurement.View
+{
+    class DivinationCardExplicitModsToFormattedRunConverter : IValueConverter
+    {
+        public object Convert(object value, Type targetType, object parameter, System.Globalization.CultureInfo culture)
+        {
+            var mod = value as String;
+
+            var paragraph = new Paragraph();
+
+            var runs = new List<Run>();
+
+            var count = 0;
+            
+            foreach (Match part in Regex.Matches(mod, "<(?<type>.*?)>\\{(?<text>.*?)\\}"))
+            {
+                switch (part.Groups["type"].Value)
+                {
+                    case "default":
+                        runs.Add(new Run(part.Groups["text"].Value + (count > 0 ? " " : "")) { Foreground = (SolidColorBrush)new BrushConverter().ConvertFrom("#7f7f7f") });
```

Comment:
Maybe consolidate and standardise the Hexcodes for each item.

Have a look at "ItemToColorBrushConvertor.cs"

Consolidating the "Colors" in the App.Xaml as StaticResources would be the way to go.